__tests__/app.test.js

```diff
@@ -84,6 +84,47 @@ describe("GET /api", () => {
   });
 });
 
+describe("GET /api/articles/:article_id", () => {
+  test("STATUS 200: responds with a correct article object", () => {
+    const expectedOutput = {
+      article_id: 1,
+      title: "Living in the shadow of a great man",
+      topic: "mitch",
+      author: "butter_bridge",
+      body: "I find this existence challenging",
+      created_at: "2020-07-09T20:11:00.000Z",
+      votes: 100,
+      article_img_url:
+        "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+    };
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        const article = response.body.article;
+        expect(article).toEqual(expectedOutput);
+      });
+  });
+  test("STATUS 400: returns an error when passed non-existent but valid article_id", () => {
+    return request(app)
+      .get("/api/articles/9999")
+      .expect(400)
```

Comment:
this should be a 404 - it's perfectly reasonable to assume there could be 9999 articles on a very popular forum/social media page.